command/title_body_survey.go

```diff
@@ -23,7 +23,11 @@ const (
 	CancelAction
 )
 
-func confirm() (Action, error) {
+var SurveyAsk = func(qs []*survey.Question, response interface{}, opts ...survey.AskOpt) error {
+	return survey.Ask(qs, response, opts...)
+}
+
+var ConfirmSubmission = func() (Action, error) {
```

Comment:
Is this explicitly a `var` so it can be overriden? I don't see us overriding this at the moment, but do we plan to in the future? Otherwise, can it be a normal func?

Reply:
oops, good catch. I had intended to override it and then ended up not doing it.